src/randoop/operation/PublicFieldParser.java

```diff
@@ -75,8 +74,10 @@ public PublicField parse(String s) throws OperationParseException {
     }
 
 
-    Class<?> type = Reflection.classForName(typeName,true);
-    if (type == null) {
+    Class<?> type;
+    try {
+      type = TypeNames.getTypeForName(typeName);
+    } catch (ClassNotFoundException e) {
       String msg = errorPrefix + " The type given \"" + typeName +"\" was not recognized.";
       throw new OperationParseException(msg);
```

Comment:
Does it make sense to chain the exception (pass `e` into the OperationParseException constructor)?